kratos/sources/model_part.cpp

```diff
@@ -1615,7 +1615,7 @@ void ModelPart::RemoveGeometry(
 
     for (SubModelPartIterator i_sub_model_part = SubModelPartsBegin();
         i_sub_model_part != SubModelPartsEnd();
-        i_sub_model_part++)
+        ++i_sub_model_part)
```

Comment:
even though this makes sense, please consider opening a separate PR for this.

Imagine that for some strange reason this breaks sth, then tracking it down will be much harder if done on the side in an unrelated PR

Reply:
This is just concerning the geometry container which was added recently. We don't even know if this is the other one would fail...